exporter/awsemfexporter/metric_translator.go

```diff
@@ -55,15 +57,29 @@ type rateState struct {
 	timestamp int64
 }
 
+// MetricInfo defines
+type MetricInfo struct {
+	Value  interface{}
+	Unit   string
+}
+
+// GroupedMetric defines
```

Comment:
Can you add more info here? Something like
```suggestion
// GroupedMetric defines a set of metrics with the same namespace, timestamp, and labels.
```